docs/VnVPlan/VnVPlan.tex

```diff
@@ -69,141 +69,157 @@ \section{Symbols, Abbreviations, and Acronyms}
+    \item Design Verification Plan: plans for design verification
+    \item Verification and Validation Plan Verification Plan: plans for VnV verification
+    \item Implementation Verification Plan: dynamic and static tests for verification of the implementation
+    \item Automated Testing and Verification Tools: tools and metrics to be used for testing and verification of the implementation
+    \item Software Validation Plan: plans for verification of the software requirements
+\end{itemize}
 
 \subsection{Verification and Validation Team}
-
-\wss{Your teammates.  Maybe your supervisor.
-  You should do more than list names.  You should say what each person's role is
-  for the project's verification.  A table is a good way to summarize this information.}
+\begin{table}[htbp]
+    \centering
+    \begin{tabularx}{\textwidth}{|>{\hsize=.36\hsize}X|>{\hsize=1.64\hsize}X|}
+        \hline
+        \textbf{Name} & \textbf{Role}\\
+        \hline
+        Allan Fang & Developer: \\
+        \hline
+        Jason Nam & Developer: \\
+        \hline
+        Ryan Yeh & Developer: \\
+        \hline
+        Sawyer Tang & Developer: \\
+        \hline
+    \end{tabularx}
+    \caption{VnV Team}
+    \label{tab:vnvteam}
+\end{table}
 
 \subsection{SRS Verification Plan}
-
-\wss{List any approaches you intend to use for SRS verification.  This may include
-  ad hoc feedback from reviewers, like your classmates, or you may plan for 
-  something more rigorous/systematic.}
-
-\wss{Maybe create an SRS checklist?}
+The design documentation will be verified with an informal walk-through by stakeholders (individuals who are members of low-income households in Hamilton and students from McMaster University) and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The inspection will address the following issues and be tracked by the checklist:
+\begin{itemize}
+    \item Correctness: the SRS accurately represents the software expectations
+    \item Ambiguity: the requirements and contents convey an unambiguous meaning
+    \item Completeness: the SRS covers all necessary functional and non-functional requirements
+    \item Verifiability: the requirements are testable and quantifiable
+    \item Traceability: the requirements have clear source references
+    \item Clarity: the document is written in a clear, concise, and unambiguous manner
+\end{itemize}
 
 \subsection{Design Verification Plan}
-
-\wss{Plans for design verification}
-
-\wss{The review will include reviews by your classmates}
-
-\wss{Create a checklists?}
+The SRS will be verified with an informal walk-through by stakeholders (individuals who are members of low-income households in Hamilton and students from McMaster University) and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The review will address the following issues and be guided by the checklist:
```

Comment:
I think this should be Design Verification instead of SRS. Also, are you sure we do an informal walkthrough here? I would think that we would do a walkthrough as a team with the developers since this is more of a technical aspect that not every stakeholder would know. We can also include here peer reviews from our classmates since we do get feedback from another team.

You should also mention that the design will be referenced with the SRS (link the SRS as seen on line 120) in addition to the checklist to verify requirements are met by the design.

Reply:
srs is typo, team does inspection bc we have expertise, peer review included in walkthrough, checklist includes srs

Reply:
stakeholders should be involved bc this is almost a usability + requirements thing